V4 backports

XhmikosR opened the pull request.

Johann-S committed:
updated babel

Brianmanden committed:
Update collapse.md (#29025)

Corrected minor spelling error.

sunn-e committed:
Added link to creative commons license (#29032)

The previous link is dead one.

alcarazzam committed:
#29034 Fix broken links (#29035)

XhmikosR commented:
@Johann-S @patrickhlauke it seems browser tests fail for v4-dev here in this branch.

patrickhlauke commented:
sigh...ok, i'd say don't backport that change, as i see all other tests there use the `trigger` method. wondering if in this particular case the `trigger` does bubble properly (compared to when i tried to use that method of firing key events in unit tests for the work on v5 aria tabs and the tests were failing/key events weren't propagating properly).

the original change is still a valid change for the v5 branch, as there we don't test IE11, and there the unit test uses the `dispatchEvent` method throughout.

XhmikosR commented:
Alright, reverted the patch. BTW we do test IE11 on master.

patrickhlauke commented:
> Alright, reverted the patch. BTW we do test IE11 on master.

so how can the exact same test fail in QUnit here but not there? (and it failed not in a "the test doesn't pass" way, but rather an "object doesn't support this" more fundamental problem with actual QUnit/the command used way)

XhmikosR commented:
That I don't know. I will try again tomorrow just in case it was a one-time thing.

XhmikosR commented:
OK, the test failures persist.

Johann-S commented on js/tests/unit/dropdown.js:
```diff
@@ -84,10 +84,11 @@ $(function () {
-    $button.trigger('focus').trigger($.Event('keydown', {
-      which: 27
-    }))
+    var keydown = new Event('keydown')
```
using `new Event` isn't supported on IE 10 and IE 11, you should use jQuery event system

XhmikosR replied:
I've already dropped the patch from this branch. If @patrickhlauke wants, we should have a separate PR.

Brianmanden committed:
Updated sentence (#29051)

Checkboxes and radios use are built to support HTML-based form

=>

Checkboxes and radio buttons support ...

wackerm committed:
Fix breaking layout breaking error in code (#28328)

Change `word-break: break-word;` to `word-wrap: break-word;`

XhmikosR committed:
Fix bootstrap-themes.png. (#29061)

For some reason the old image was bad quality.

XhmikosR committed:
Update devDependencies.
